08.ls_object/lib/long_style.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+class LongStyle
+  MARGIN = 1
+  HALF_A_YEAR = 182
+
+  def initialize(files)
+    @files = files
+  end
+
+  def draw
+    width = calc_width
+
+    print_total
+    @files.each do |file|
+      show_content(file, width)
+    end
+  end
+
+  private
+
+  def show_content(file, width)
+    print_type(file)
+    print_mode(file)
+    print file.nlink.to_s.rjust(width[:nlink]), "\s"
+    print file.uid.name.ljust(width[:uid])
+    print file.gid.name.ljust(width[:gid])
+    print file.size.to_s.rjust(width[:file_size]), "\s"
+    print_timestamp(file)
+    print file.path, "\n"
+  end
+
+  def print_total
+    blocks = @files.sum { |file| file.stat.blocks }
```

Comment:
```suggestion
    blocks = @files.sum { |file| file.blocks }
```

のようにすれば`stat`を外部に公開しなくて良くなりますね。
`nlink`の実装との一貫性も出てきます。

Reply:
`nlink`に合わせました